app/controllers/avo/search_controller.rb

```diff
@@ -121,8 +126,6 @@ def apply_search_metadata(records, avo_resource)
       end
     end
 
-    private
```

Comment:
is this required?

Reply:
My bad adding that on past commits, there is already a `private` scope on L16 on that file. This one can be safely removed

Reply:
Good spotting!